Sources/MongoSwift/ReadWriteConcern.swift

```diff
@@ -33,8 +33,8 @@ public class ReadConcern: Codable {
         return mongoc_read_concern_is_default(self._readConcern)
     }
 
-    /// Initialize a new `ReadConcern` from a `ReadConcernLevel`.
-    public convenience init(_ level: ReadConcernLevel) {
+    /// Initialize a new `ReadConcern` from a `ReadConcern.Level`.
```

Comment:
If a user imports `MongoSwift`, I assume `ReadConcern` is then accessible and they will need to access `Level` as `ReadConcern.Level`, correct?

Did we have any tests that needed to be updated, or are those all using the `init()` from a raw string value?

Reply:
Yes, that's correct. If they ever needed to explicitly use the type, they would have to use the full `ReadConcern.Level`.

However, for the vast majority of times users are interacting with this type (passing it into a `ReadConcern` initializer), the compiler just infers the type and its name is not needed. So both before and after, 
```swift
let rc = ReadConcern(.majority)
```
works, and this is how we're interacting with this type throughout the tests. 

The only time you'd need the type name would be if you were saving a level in a variable somewhere - for example
```swift
let level: ReadConcernLevel = .majority
```
would now be
```swift
let level: ReadConcern.Level = .majority
```

We don't do that anywhere though, so no changes needed.